templates/program/nph/order/collect.html.twig

```diff
@@ -58,6 +58,9 @@
                             {{ form_errors(orderCollectForm[nphSample.sampleCode]) }}
                             {% if nphSample.modifyType == constant('UNLOCK', nphSample) %}
                                 <p class="text-warning"><i class="fa fa-exclamation-triangle"></i> Unlocked for editing</p>
+                            {% elseif nphSample.modifyType == constant('EDITED', nphSample) and nphSample.finalizedTs is not empty %}
```

Comment:
Since we are display the finalizedTs for both cases, you could just check the modified type to display the status text.
```
     {% elseif nphSample.finalizedTs is not empty %}
                  <p class="text-success"><i class="fa fa-check-circle"></i> 
                  {{ nphSample.modifyType == constant('EDITED', nphSample) ? 'Aliquoted and Finalized' : 'Finalized' }}
                  {{ nphSample.finalizedTs|date('n/j/Y g:ia', app.user.timezone) }}</p>
    {% endif %}
```